bin/publish_helm_chart

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env bash
+
+set -evuo pipefail
+
+echo "Installing helm"
+
+curl https://raw.githubusercontent.com/kubernetes/helm/master/scripts/get > get_helm.sh
+chmod 700 get_helm.sh
+./get_helm.sh --version "${HELM_VERSION}"
+helm init --client-only
+
+echo "Publishing helm chart"
+
+output="$(cd helm; helm package fiaas-skipper)"
+echo $output
+package=`expr "$output" : 'Successfully packaged chart and saved it to: \(.*\)'`
+echo $package
+
+git clone https://github.com/fiaas/helm helm-repo
+mv $package helm-repo/
+cd helm-repo/
+helm repo index . --url https://fiaas.github.io/helm/
+git add .
+git commit -a -m "Release fiaas-skipper"
```

Comment:
I just thought of something that might improve traceability: if we add `${TRAVIS_REPO_SLUG}@${TRAVIS_COMMIT}` to the commit message here, I think that should render as a link to the commit in the skipper repo that triggered the build that built the release (see https://help.github.com/articles/autolinked-references-and-urls/). What do you think?